other-docs/guides/upgrading/v12.md

```diff
@@ -0,0 +1,54 @@
+---
+order: 12
+---
+# Upgrading to v12
+
+_If you are migrating from WordPress to Altis, check out the [migrating guide here](../migrating-from-wordpress.md) first._
+
+To upgrade to Altis v12, edit your `composer.json` and change the version constraint for `altis/altis` and any local environment modules to `^12.0.0`.
+
+```json
+{
+	"require": {
+		"altis/altis": "^12.0.0"
+	},
+	"require-dev": {
+		"altis/local-chassis": "^12.0.0",
+		"altis/local-server": "^12.0.0"
+	},
+	"config": {
+		"platform": {
+			"php": "8.0"
+		}
+	}
+}
+```
+
+Once you have made these changes run `composer update` and then run the `wp altis migrate` command:
+
+```sh
+# For cloud environments
+wp altis migrate
+
+# For local server
+composer server cli -- altis migrate
+
+# For local chassis
+composer chassis exec -- wp altis migrate
```

Comment:
Hmm, did we break chassis somewhere along the line (or is it only the M1 thing)?